jsettlers.mapcreator/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+encoding//src/jsettlers/mapcreator/localization/labels_de.properties=UTF-8
+encoding//src/jsettlers/mapcreator/localization/labels_en_US.properties=UTF-8
+encoding//src/jsettlers/mapcreator/localization/labels_ru.properties=UTF-8
```

Comment:
Is it possible to specify that all files in /localization have to be UTF-8 encoded? This would make it save for the future. (Same with commons project).